docs/conf.py

```diff
@@ -27,6 +27,17 @@
 html_static_path = ["static"]
 html_css_files = ["custom.css"]
 
+extensions = ["sphinx_copybutton"]
+
+# -- sphinx-copybutton configuration ----------------------------------------
+## sets up the expected prompt text from console blocks, and excludes it from
+## the text that goes into the clipboard.
+copybutton_exclude = '.linenos, .gp'
+copybutton_prompt_text = ">> "
```

Comment:
Does this imply that we need to make sure that every line that is copied has `>> ` at the beginning? Or does this simply mean that copybutton will attempt to remove a `>> ` if it sees one?

Reply:
Ok it looks like it's kinda both. https://sphinx-copybutton.readthedocs.io/en/latest/use.html#strip-and-configure-input-prompts-for-code-cells

My understanding is that it will only copy lines that have `>> ` in this case, and when it does copy those, it will also remove the `>> ` token. Pleas correct me if I'm way off.

Reply:
My read (and what I've observed by painfully clicking the copy button on every code block in the PPT docs) is that it will show a copy button for EVERY code block.

If the code block is a "console" type, it will try to suppress (exclude) the "prompt" text, which is likely "$" or ">>>". We have been using just two ">> " for prompt text, and I think it looks nice, so I manually added it as a prompt text.